pyiron_atomistics/atomistics/structure/neighbors.py

```diff
@@ -182,26 +235,25 @@ def _get_wrapped_positions(self, positions, distance_buffer=1.0e-12):
         x = np.array(positions).copy()
         cell = self._ref_structure.cell
         x_scale = np.dot(x, np.linalg.inv(cell))+distance_buffer
-        x[...,self._ref_structure.pbc] -= np.dot(np.floor(x_scale),
-                                                 cell)[...,self._ref_structure.pbc]
+        x[...,self._ref_structure.pbc] -= np.dot(
+            np.floor(x_scale), cell
+        )[...,self._ref_structure.pbc]
         return x
 
+    @deprecate(allow_ragged="use mode instead.")
     def _get_distances_and_indices(
         self,
         positions=None,
+        mode='filled',
         allow_ragged=None,
         num_neighbors=None,
         cutoff_radius=np.inf,
         width_buffer=1.2,
     ):
+        if allow_ragged is not None:
```

Comment:
I would then also shift this check to `get_distances` and `get_indices` accordingly.

Reply:
Oh ok